lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,161 @@ fields:
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
+    description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+    action: update
+    example: |
+      {
+        "questionnaire": "https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041"
+      }
+
+  - name: status
+    required: 'Yes'
+    type: code
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status
+    description: The position of the questionnaire response within its overall lifecycle. Must be 'in-progress'.
+    action: update
+    example: |
+      {
+        "status": "in-progress"
+      }
+  
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject
+    description: The subject of the questionnaire response. This could be a patient, organization, practitioner, device, etc. This is who/what the answers apply to, but is not necessarily the source of information.
+    action: update
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12508041"
+        }
+      }
+
+  - name: authored
+    required: 'No'
+    type: dateTime
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.authored
+    description: The date and/or time that this set of answers were last changed.
+    action: update
+    example: |
+      {
+        "authored": "2019-10-25T14:57:43Z"
+      }
+
+  - name: item
+    required: 'Yes'
+    type: List of BackboneElement
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item
+    description: A group or question item from the original questionnaire for which answers are provided. Only answer value types of String, Quantity, and Coding are supported.
+    action: update
+    example: |
+      {
+        "item": [
+          {
+            "linkId": "93",
+            "text": "Tobacco",
+            "item": [
+              {
+                "linkId": "93-123",
+                "text": "Number of Years:",
+                "answer": [
+                  {
+                    "valueQuantity": {
+                      "value": 20
+                    }
+                  }
+                ]
+              }
+            ] 
+          }
+        ]
+      }
+
+  - name: item.linkId
+    required: 'Yes'
+    type: string
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId
+    description: The item from the Questionnaire that corresponds to this item in the QuestionnaireResponse resource.
+    action: update
+    example: |
+      {
+        "linkId": "93"
+      }
+
+  - name: item.text
```

Comment:
we dont validate this or send it down so remove

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/6b1e581f4f01eb2d8a1a69a8a0f857da78c0fcc3